src/py/pyodide/__init__.py

```diff
@@ -53,27 +53,31 @@
 __version__ = "0.20.0"
 
 __all__ = [
-    "open_url",
-    "eval_code",
-    "eval_code_async",
     "CodeRunner",
-    "find_imports",
-    "JsProxy",
+    "ConversionError",
+    "IN_BROWSER",
```

Comment:
This is the only added item, I also sorted the list.

Reply:
I don't think we want `IN_BROWSER` as part of the public API since it should always be `True` except when testing. Also the name is misleading because it is `True` in Node.

Reply:
I guess `micropip` should be updated to import it from `pyodide._core`

Reply:
Done